src/components/Display/Display.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import dateFormatter from '../../utils/dateFormatter.js'
+
+export default function Display({content}) {
+    return (
+        <section>
+            <p>{content ? dateFormatter(content) : 'Pick a date'}</p>
```

Comment:
great use of a helper function (I've noticed more of these in your code which makes for very easy reading!)